csxtools/utils.py

```diff
@@ -1,9 +1,6 @@
 import numpy as np
 import time as ttime
-try:
-    from databroker import get_images
-except ImportError:
-    from dataportal import get_images
+from databroker import get_images
```

Comment:
That is because `!which conda` gives

```
/GPFS/xf23id/users/swilkins/miniconda3/bin/conda
```
It must be my account, thinking about it the complaint was from @vivekthampy and he probably has the same issue. 

Therefore lets just merge this.